### PR TITLE
Added mode feature to fix issue of newly added elements

### LINKED ISSUE #20: Detect newly added elements and highlight them as well
when we wrap the cart for example in `ChangeHighlight` it highlights the total price as required but the newly added items are not highlighted, and this is important to be highlighted as well

So the expected result is to detect new elements inside the wrap and highlight them as well.

### PR DESCRIPTION
**Pull request by medhatdawoud**
- Add mode feature
- add readme file changes

fix #20 #21

### CODE REVIEW

File: packages/react-change-highlight/src/ShadowChild.tsx [hunk 1]
```diff
@@ -35,23 +37,26 @@ const ShadowChild: React.FC<Props> = ({
   }
 
   useEffect(() => {
-    if (initialMount?.current) {
+    if (!newlyAddedOnly && initialMount?.current) {
```
Comment (mohamedGamalAbuGalala) [inline packages/react-change-highlight/src/ShadowChild.tsx:40]: :clap: :clap: 
I couldn't figure it out back then, but it seems very silly :sweat_smile:

File: packages/react-change-highlight/src/Util/setHighlight.ts [hunk 2]
```diff
@@ -16,7 +17,10 @@ const setHighlight = (
 
   const element = child.ref.current;
 
-  if (!isInitial && element?.getAttribute(defaults.HIGHLIGHT_UNIQUE_ID) === uniqueId) {
+  if (
+    (newlyAddedOnly ? isInitial : !isInitial) &&
```
Comment (mohamedGamalAbuGalala) [inline packages/react-change-highlight/src/Util/setHighlight.ts:21]: :clap: :clap:

**Review by mohamedGamalAbuGalala — COMMENTED** [review]
Great job :clap: :clap: 
I wish I could take some time for contributing again.

**Comment by mohamedGamalAbuGalala** [comment]
https://codesandbox.io/s/adoring-violet-yswpi

**Comment by medhatdawoud** [comment]
Nope, this is not the case I was talking about, in your case the element is already there but you changed it's value, check the todo example in the examples folder to know what I meant

**Pull request merged**